sdv/multi_table/hma.py

```diff
@@ -398,6 +399,10 @@ def _augment_table(self, table, tables, table_name):
                     self.extended_columns[child_name][column] = FloatFormatter(
                         enforce_min_max_values=True)
                     self.extended_columns[child_name][column].fit(extension, column)
+                if len(extension.columns) > 0:
+                    if table_name not in self.parent_extended_columns:
+                        self.parent_extended_columns[table_name] = []
+                    self.parent_extended_columns[table_name] = list(extension.columns)
```

Comment:
I think this should be an append, otherwise if you have more than 1 child the 2nd child will overwrite the extended columns from the first

Reply:
Fixed with using extend:
https://github.com/sdv-dev/SDV/pull/2068/commits/e38bc2a2c9eabcd91329166617dea2d3266696f7#diff-c18ffdc4352bb1e367eaefa2e851a0da762907b7bf727f00bc6094446c889beeR419